src/app/poll/mobile-poll-table.component.scss

```diff
@@ -15,6 +15,11 @@
   width: 45px;
   height: 100%;
   vertical-align: middle;
+  outline-offset: 0;
+}
+
+.btn.btn-trash {
+  @include focus-outline;
```

Comment:
könnte man hieraus nicht eine globale Klasse `focus-outline` machen? Wär doch eigentlich praktisch

Reply:
Hab ich nun so umgesetzt, danach stand der mixin alleine und wurde zum einzigen include verschoben.

Reply:
Achso, so hab ich das gar nicht gemeint gehabt :D das Mixin zu haben wäre auch nicht schlecht gewesen, weil man es dann in anderen css-Regeln verwenden kann 🙈 Ich dachte nur es wäre praktisch dann zusätzlich eine globale Klasse zu haben, die das mixin nur noch anwendet. Bin mir gerade nicht sicher wie gut ich das so finde wie es jetzt ist, sorry für die Verwirrung :D

Reply:
Kommt am Ende ja aufs Gleiche hinaus. Welche Version ist dir lieber, mit oder ohne mixin?

Reply:
Reverted und Klasse lediglich hinzugefügt.